### ISSUE TITLE
we should regularly check to ensure that our jsonld is compliant with schema.org

### ISSUE THREAD

**Issue by devincowan** [hydroshare/hydroshare#4603]
**Description**
Schema.org updates their schemas (frequently?). We should make sure that we are still compliant.

Steps to reproduce:
1. Inspect the #schemaorg tag in the `<head>`

**Additional information**
Ideally we would do this on a regular recurring basis. Maybe quarterly?

**Comment by horsburgh** [comment]
@devincowan - this is by design. According to Schema.org, the citation element should contain "A citation or reference to *another* creative work, such as another publication, web page, scholarly article, etc.

https://schema.org/citation

It may seem counterintuitive, but we are using the citation element as Schema.org has specified it.

**Comment by devincowan** [comment]
got it thanks @horsburgh, I see now...

**Comment by horsburgh** [comment]
I know you closed this, but I think it's good to be looking closely at our Schema.org implementation. There are elements that are changing with Schema.org. In particular, I think I have recently seen some movement related to how funding information is expressed. It's important to make sure we keep our implementation current.

**Comment by martinseul** [comment]
@devincowan this is a good idea to keep track of this as it is changing.  There is a ESIP group https://wiki.esipfed.org/Schema.org_Cluster  involved in this  and they have a Github repo  https://github.com/ESIPFed/science-on-schema.org/blob/master/README.md maybe something to keep an eye out for. Do you have other suggestions on what else to do to keep informed? Can we have an automated test for it?

**Comment by horsburgh** [comment]
What would be really nice is if the ESIP group (or anyone else) had some validation tools that we could use. Right now, you sort of embed your Schema.org metadata in your pages and then cross your fingers and hope the major search providers are crawling it effectively.  I haven't checked recently to see if they have something like that.

**Comment by martinseul** [comment]
google has one https://developers.google.com/search/docs/advanced/structured-data

**Comment by horsburgh** [comment]
Yeah - the is useful. It will extract the schema.org metadata, present you with a view of it, and tell you if you have errors or warnings. I just tried it again and was happy that it shows 0 errors and 0 warnings for a HydroShare landing page. What it doesn't tell you is - does your Schema.org metadata conform to the ESIP SOSO recommendations or whether the way you have implemented the elements is ideal for cataloging.

**Comment by hyi** [comment]
Here is another validation tool from DataONE https://github.com/datadavev/sonormal which I used for validation when implementing schema.org for HydroShare.
